packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -4,10 +4,16 @@
       <div v-if="link.path">
         <router-link
           :to="link.path"
-          exact
```

Comment:
Are we intending to drop the `exact` match requirement here?

Reply:
yes, since we using dynamically changed hash routs `exact` would not work, it calculates absolute path only